src/parser/parser.yy

```diff
@@ -2903,10 +2903,16 @@ show_sentence
          $$ = new ShowEdgesSentence();
     }
     | KW_SHOW KW_TAG KW_INDEXES {
-         $$ = new ShowTagIndexesSentence();
+         $$ = new ShowTagIndexesSentence(new std::string(""));
+    }
+    | KW_SHOW KW_TAG KW_INDEXES KW_BY name_label {
```

Comment:
What's the meaning of the `by label` ? And you should add case in tck with this.